Las/Utils/Filetype.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Lucidtech.Las.Utils
+{
+    /// <summary>
+    /// Help determine the type of a file, inspired by pythons <c>imghdr.what()</c>.
+    /// </summary>
+    public static class FileType
+    {
+        /// <summary>
+        /// Tests the type of file.
+        /// </summary>
+        /// <param name="fileName"> The name of the file that is to be classified </param>
+        /// <returns> The name of the file type that matches, or an empty string if the file does not match </returns>
+        public static string WhatFile(string fileName)
```

Comment:
Looks to me like returned filetype can be an empty string "" here. I suggest throwing an exception if filetype ends up being the empty string. Also, if possible I would suggest initializing filetype with null instead of empty string, as it's semantically more correct imo.